src/conda_constructor/uninstall.py

```diff
@@ -171,7 +182,13 @@ def _run_conda_init_reverse(for_user: bool, prefix: Path, prefixes: list[Path]):
     # That function will search for activation scripts in sys.prefix which do no exist
     # in the extraction directory of conda-standalone.
     run_plan(plan)
-    run_plan_elevated(plan)
+    try:
+        run_plan_elevated(plan)
+    except Exception:
+        logger.error(
+            "Could not revert some shell profiles because they require elevated privileges. "
+            "Check the output for lines with `needs sudo` and edit those files manually."
+        )
```

Comment:
```suggestion
    except Exception as exc:
        logger.error(
            "Could not revert some shell profiles because they require elevated privileges. "
            "Check the output for lines with `needs sudo` and edit those files manually.",
            exc_info=exc,
        )
```